simplified-app-shared/src/main/res/values/strings.xml

```diff
@@ -1,9 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <!-- Action/Navigation Strings -->
+  <string name="nav_confirm">Confirm</string>
+  <string name="nav_done">Done</string>
+  <string name="nav_next">Next</string>
 
   <!-- EULA strings -->
   <string name="eula_disagree">Disagree</string>
   <string name="eula_agree">Agree</string>
+  <string name="eula_checkbox">I agree to the terms of the End User License Agreement.</string>
+
+  <!-- Age Verification strings -->
+  <string name="age_verification_title">Age Verification</string>
+  <string name="age_verification_question">You must be 13 years of age or older to download a book from the collection.\nHow old are you?</string>
+  <string name="age_verification_changed">If you are under 13, all content downloaded to My Books will be removed.\nHow old are you?</string>
```

Comment:
all content, or all YA content?

Reply:
If this comes back as true, it looks like it calls reloadCatalog with a true parameter for deleting the books in the catalog.

I don't know how that code works on the back end.

Reply:
We serve two completely different OPDS feeds based on the age, so I guess "all content". Any time you decide to change your age in settings, the confirmation informs the user that they will lose their current content and start over with the new feed.